### ISSUE TITLE
Request and a question

### ISSUE THREAD

**Issue by AhoyThere** [lostdusty/gualto#1]
Hello there,

I recently discovered not only Cobalt Tools but your GUI tool as well. Thank your for coding such a much needed GUI.

I do however have a request and a question. First of all, would it be possible to have Gualto remember and even auto-download to the last used folder? Having to manually select it every time kinda sucks.

And my question is, where are Gualto's settings saved? Any way to revert them to default?

Thanks in advance

**Comment by lostdusty** [comment]
Hello, thanks for your input!

Yes, it is possible! I've discussed this question on the Fyne community, and I'm waiting their PR to be added to the project... in other words, I can add that to Gualto.

For the next question, depends on your operating system, but its located either on the internal application data folder (like Android and iOS), or somewhere like `%localappdata%` (on Windows). If needed I can also make a button to reset the settings to default.

**Comment by AhoyThere** [comment]
Hey there again.

Thank you for your reply. I can't wait for the "remember last used folder" option. Thank you in advance for adding it in the future.

When it comes to my question about resetting Gualto options I was indeed talking about Windows, Windows 10 to be exact. I don't know if a button to reset them is needed but I for one can't find a way to do it. I can't find a settings file, neither in the folder from where I run Gualto nor in Appdata or the Temp sub-directory.

**Comment by lostdusty** [comment]
On Windows, it's under `%appdata%\Roaming\Fyne\com.lostdusty.gualto` just delete the file. If you don't mind, is there any reason you want to reset the app settings? Just a little curious ;)

**Comment by AhoyThere** [comment]
Hey,

thank you once again for your reply.

Weirdly enough I couldn't find it under %localappdata%\Fyne\ but with the knowing it should be called "Fyne" I searched a bit deeper. %localappdata% points to AppData\Local but I couldn't find it there. Instead I found it under AppData\Roaming where I was able to delete it just as you wrote. Thank you.

There reason for my request to delete the setttings is simple. I ignored the warning in Gualto's settings and changed the URL for the custom instance setting. And when I wanted to change it back to the original URL I couldn't remember what it was. I know, stupid me.

Oh and just in case you want to know why I changed it in first place, that's because the downloads stopped working. I couldn't find a reason so I thought it might be related to the instance URL. After some trying around and by counter-checking the Cobal Tools own website I however found out that the Cobalt Tools website/service itself just stops working several times a day, sometimes for hours, and therefore Gualto couldn't work..  :)
